OVS_Bonding/README.md

```diff
@@ -0,0 +1,57 @@
+# <u>Setting up network bonding on specific interfaces during the ignition process</u>
+
+## Overview 
+This guide explains how to bond specific NICs during the initial OpenShift installation with OVS and supplies the Ignition and scripts to achieve this.
+
+## Important
+- All raw files and scripts are located in the RewFiles folder.
+- This guide describes the customisations needed for installation when using a PXE server.
+<br>
+
+## Prerequisites
+- PXE server with ability to see kernel arguments.
```

Comment:
not sure if this is what you're after

```suggestion
- PXE server with ability to set kernel arguments.
```

Reply:
Sounds right and more to the point , Fixed